examples/todomvc/routes.js

```diff
@@ -0,0 +1,11 @@
+import { StateInfoConfig } from 'navigation'
+import { SHOW_ALL } from './constants/TodoFilters'
+
+function configureStateInfo() {
+  StateInfoConfig.build([
+    { key: 'todomvc', initial: 'app', states: [
+      { key: 'app', route: '{filter?}', defaults: { filter: SHOW_ALL }, trackCrumbTrail: false }
+    ] }   
+  ])
+}
```

Comment:
Builds the route configuration. There’s one route with an optional filter parameter that has a default value of 'all'